src/sage/rings/polynomial/polynomial_quotient_ring.py

```diff
@@ -1260,12 +1260,15 @@ def polynomial_ring(self):
 
     cover_ring = polynomial_ring
 
-    def random_element(self, *args, **kwds):
+    def random_element(self, *args, degree=None, **kwds):
         """
```

Comment:
Is there a reason why you placed the optional `degree` argument after `*args`, instead of in front as it is done for `.random_element()` in the PolynomialRing_general class and the method call a few lines below?